app/src/main/res/layout/layout_attachments.xml

```diff
@@ -28,46 +28,41 @@
         android:clipToPadding="false"
         android:orientation="horizontal"
         android:paddingHorizontal="@dimen/alternativeMargin"
-        android:paddingBottom="@dimen/marginStandardMedium"
+        android:paddingBottom="@dimen/marginStandardSmall"
         app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_goneMarginTop="@dimen/marginStandardSmall"
         tools:listitem="@layout/item_attachment" />
 
-    <ImageView
-        android:id="@+id/attachmentsIcon"
-        android:layout_width="@dimen/mediumIconSize"
-        android:layout_height="@dimen/mediumIconSize"
-        android:layout_marginStart="@dimen/marginStandardMedium"
-        android:importantForAccessibility="no"
-        android:src="@drawable/ic_attachment"
-        app:layout_constraintBottom_toBottomOf="@id/attachmentsSizeText"
+    <LinearLayout
+        android:id="@+id/attachmentsInfo"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingHorizontal="@dimen/marginStandardMedium"
+        android:clickable="true"
+        android:focusable="true"
```

Comment:
I'm not sure this two attributes are needed, I think the `android:background="?attr/selectableItemBackground"` is enough 
Maybe confirm this with @LunarX